spec/jobs/mailchimp_signup_job_spec.rb

```diff
@@ -6,9 +6,7 @@
 
   it 'enqueues the job when provided with the correct email' do
     expect(Mailchimp).to receive(:signup).with('fake@email.name', mailchimp_list.id)
-
     MailchimpSignupJob.perform_now('fake@email.name', mailchimp_list)
-
   end
-  
+    
```

Comment:
Ideally this whitespace change should be removed but I'm not going to hold this up for that.